assignment/tool/src/webapp/js/assignments.js

```diff
@@ -852,6 +852,24 @@ ASN.toggleAutoAnnounceOptions = function(checked){
     }
 };
 
+ASN.toggleAutoAnnounceEstimate = function(checked){
+    var obEstimate = document.getElementById("checkobestimate");
+    var timeEstimate = document.getElementById("inputtimestimate");
+    var timeEstimateInput = document.getElementById("new_assignment_input_add_time_estimate");
+    var obEstimateCheck = document.getElementById("new_assignment_check_add_ob_estimate");
+    if(checked){
+    	obEstimate.style.display="block";
```

Comment:
obEstimate && (obEstimate.style.display = "block");

Reply:
If you combine the element's existence check and modification into truthy type statement like this it makes the code more reliable.

Reply:
method has been refactored.